be2-scala/src/test/scala/ch/epfl/pop/pubsub/graph/DbActorSuite.scala

```diff
@@ -41,15 +41,13 @@ class DbActorSuite() extends TestKit(ActorSystem("myTestActorSystem"))
   val dbActorRef: AskableActorRef = system.actorOf(Props(DbActor(pubSubMediatorRef, DB_TEST_FOLDER)), "DbActor")
 
   override def afterAll(): Unit = {
-    // Stops the test actor system
-    TestKit.shutdownActorSystem(system)
+     TestKit.shutdownActorSystem(system, verifySystemShutdown = true)
 
     // Deletes the test database
     val directory = new Directory(new File(DB_TEST_FOLDER))
     directory.deleteRecursively()
```

Comment:
The call to deleteRecursively() never succeeds.

Reply:
is this now fixed with the new addition to DbActor?

Reply:
Yes